### ISSUE TITLE
Request: (GNU) Savannah

### ISSUE THREAD

**Issue by MrWils** [simple-icons/simple-icons#1167]
**Name:** (GNU) Savannah
**Website:** https://savannah.gnu.org/
**Official resources for icon and color:**
- https://savannah.gnu.org/images/Savannah.theme/floating.png
- https://en.wikipedia.org/wiki/GNU_Savannah#/media/File:Gnu_meditate_levitate.png

**Note:**
I couldn't find anything in an another format so both images are PNG.
I am not sure what was meant with color but I doubt that I could even link it since I couldn't find anything else as a PNG.

**Comment by ericcornelissen** [comment]
That brings up an interesting point because we already [provide GNU logos (namely _GNU_ and _GNU social_)](http://simpleicons.org/?q=gnu) which I imagine also use the GPL license - although I could not quickly confirm this....

**Comment by MrWils** [comment]
It is not a problem since:

- The GNU head is CC BY-SA 2.0 licensed. 
source: https://en.wikipedia.org/wiki/GNU#/media/File:Heckert_GNU_white.svg

- The GNU Social logo is CC0 licensed.
source: https://en.wikipedia.org/wiki/GNU_social#/media/File:GNU-social-logo.svg


The GNU Savannah would be a problem. 
It is GPLv3 licensed.

**Comment by ericcornelissen** [comment]
Thanks for investigating this further @MrWils, am I correct that in the case of a CC BY-SA 2.0 license of the GNU head we should "give appropriate credit" as per https://creativecommons.org/licenses/by-sa/2.0/ ?

**Comment by MrWils** [comment]
Yes, I think so.   
It also says that it should be under the same license.  
I am not sure if it has to be since a lot of things seem to use the GNU head.  

I found this [page](https://www.gnu.org/graphics/heckert_gnu.html) on the GNU site after a bit of searching.

> If you want to use the GNU head to link to a website run by the Free Software Foundation or the GNU project, feel free, or if you're using it in contexts talking about GNU in a supportive and accurate way, you can also do this without permission. For any other requests, please ask licensing@fsf.org for permission first.

I am pretty sure that they would be ok with this but I honestly would send them a mail to be 100% sure.

**Comment by ericcornelissen** [comment]
> ShareAlike — If you remix, transform, or build upon the material, you must distribute your contributions under the same license as the original. 

The above comes from the [CC-2.0](https://creativecommons.org/licenses/by-sa/2.0/) page, I would argue we don't "remix, transform, or build upon the material" 🤔

**Comment by MrWils** [comment]
You are correct if it is [the original SVG](https://en.wikipedia.org/wiki/GNU#/media/File:Heckert_GNU_white.svg). 
It *could* be different if someone created their own SVG but I am not sure.

**Comment by MrWils** [comment]
I assume that we are safe as well.

The project isn't promoting hate towards GNU.  
It is supportive towards GNU since the icon is available.  
The FSF usually likes supportive things.

**Comment by ericcornelissen** [comment]
Thanks for all the help @MrWils 😃
